torch/linalg/__init__.py

```diff
@@ -484,7 +484,9 @@
 Args:
     input (Tensor): the input matrix of size `(m, n)` or the batch of matrices of size `(*, m, n)`
                     where `*` is one or more batch dimensions.
-    tol (float, optional): the tolerance value. Default is ``None``
+    tol (float, Tensor, optional): the tolerance value to determine the cutoff for small singular values.
```

Comment:
"the tolerance value or values to determine..."

what happens to singular values below their corresponding tolerance?

Reply:
They're not considered in the computation of matrix rank then. It's described in the main text:
"The matrix rank is computed as the number of singular values that are greater than the specified tol threshold."
I don't think we need to duplicate this sentence here, right?

Reply:
I agree with you.